seqr/views/utils/terra_api_utils.py

```diff
@@ -238,10 +238,8 @@ def add_service_account(user, workspace_namespace, workspace_name):
     :param workspace_name: name of the workspace on AnVIL. The name will also be used as the name of project in seqr
     :return: Success: True, Fail: False
     """
-    old_acl = user_get_workspace_acl(user, workspace_namespace, workspace_name)
-    service_account = old_acl.get(SERVICE_ACCOUNT_FOR_ANVIL)
-    if service_account and not service_account['pending']:
-        return True
+    if has_service_account_access(user, workspace_namespace, workspace_name):
+        return False
```

Comment:
I would like to update the descriptions for the return in the docstring to spec `True` for successfully added and `False` for account exist.

Reply:
updated